docs/tutorials/getting_started.rst

```diff
@@ -185,23 +185,36 @@ exponential decay model of the :math:`T_1` experiment:
 Analysis Results
 ----------------
 
-The analysis results resulting from the fit are accessed with :meth:`~.ExperimentData.analysis_results`:
+The analysis results resulting from the fit are accessed with
+:meth:`~.ExperimentData.analysis_results`. If the ``dataframe=True`` parameter is passed, analysis
+results will be retrieved in the pandas :class:`~pandas:pandas.DataFrame` format:
+
+.. jupyter-execute::
+
+    exp_data.analysis_results(dataframe=True)
+
+Alternatively, analysis results can be accessed via the legacy list format:
 
 .. jupyter-execute::
 
     for result in exp_data.analysis_results():
         print(result)
 
-Results can be indexed numerically (starting from 0) or using their name. Analysis results can also be
-retrieved in the pandas :class:`~pandas:pandas.DataFrame` format by passing ``dataframe=True``:
+Individual results can be retrieved using their name or their long or short IDs, which will all
+return the corresponding :class:`AnalysisResult` object:
```

Comment:
@coruscating Oops, I just started writing some sphinx and remember I had meant to point out the missing `.` in front of `AnalysisResult` here.